smt/surrogate_models/tests/test_krg.py

```diff
@@ -88,14 +103,16 @@ def _check_derivatives(sm, xt, yt, ndim, i=10):
         dydx_predict = np.zeros(ndim)
         for j in range(ndim):
             dydx_predict[j] = sm.predict_derivatives(x_test, kx=j)[0]
-            dydx_predict[j] = sm.predict_derivatives(x_test, kx=j)[0]
-            dydx_predict[j] = sm.predict_derivatives(x_test, kx=j)[0]
-            dydx_predict[j] = sm.predict_derivatives(x_test, kx=j)[0]
+
         print(dydx_predict)
         print(yt[i, 1:])
 
         # compare results
-        np.testing.assert_allclose(yt[i, 1:], dydx_predict, atol=2e-3, rtol=1e-3)
+        if sm.options["corr"] == "squar_exp":
```

Comment:
Seen with Paul, weird test to be suppressed. Seems we are comparing true yt values and its derivatives while test function being Sphere?!

Reply:
fixed